tests/Microsoft.Identity.Test.Unit.net45/PublicApiTests/TelemetryTests.cs

```diff
@@ -93,6 +102,71 @@ public void TelemetryInternalApiSample()
             Assert.IsTrue(_myReceiver.EventsReceived.Count > 0);
         }
 
+        [TestMethod]
+        public void NoTelemetryLogicIfNoCallbackOrConfig()
+        {
+
+            _telemetryManager = new TelemetryManager(
+                _serviceBundle.Config,
+                _platformProxy,
+                null,
+                true);
+
+            var correlationId = Guid.NewGuid().AsMatsCorrelationId();
+
+            var e1 = new ApiEvent(_logger, _crypto, correlationId) { Authority = new Uri("https://login.microsoftonline.com"), AuthorityType = "Aad" };
```

Comment:
nit: I think we already have constant values for these

Reply:
I'm sure we do, but the tests in this file do not use them. It's just an URI, has no meaning to telemetry.